addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -3698,6 +3698,7 @@ export class OdooEditor extends EventTarget {
                             );
                         }
                         tab.remove();
+                        this._updateToolbar(true);
```

Comment:
Hello @visp-odoo , without any condition on it , what this fix will do is, it will activate the toolbar when i normally hit shift+tab,even though the text is not selected.
![image](https://github.com/odoo/odoo/assets/103405229/da85fe5e-10cb-4d9f-9065-da8ac5aa7b51)

Reply:
Done changes :+1: